content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-job-stories/heading4.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: nav/section
+section: Follow up
```

Comment:
Duplicates content from `heading3.md` in the same folder.

Reply:
Steve fixed.